binary/__init__.py

```diff
@@ -96,6 +96,17 @@
     'DiskTierConfig',
     'TieredStoreConfig',
     "PartitioningAttributeConfig",
+    "WanConsumerConfigHolder",
+    "WanCustomPublisherConfigHolder",
+    "WanBatchPublisherConfigHolder",
```

Comment:
DiscoveryConfigHolder is missing. I also realized we didn't regenerate the binary compatibility tests in the main PR.

Reply:
Everything is there that needs to be, as discussed.